src/components/search_bar/filters/field_value_selection_filter.tsx

```diff
@@ -246,7 +247,7 @@ export class FieldValueSelectionFilter extends Component<
   onOptionClick(
     field: string,
     value: Value,
-    checked: 'on' | 'off' | undefined
+    checked: Omit<EuiSelectableOptionCheckedType, 'mixed'> | undefined
```

Comment:
Nice use of `Omit`! It looks like `undefined` is already included in `EuiSelectableOptionCheckedType` so you should be able to do just this
```suggestion
    checked: Omit<EuiSelectableOptionCheckedType, 'mixed'>
```

Reply:
Super bizarre - I wonder why TS wasn't picking up the `| undefined` in the original type. I ended up using `?:` notation instead of the longer `| undefined`